go.sh

```diff
@@ -26,29 +26,35 @@ if [ "${LOCATIONS_CSV}" == "" ]; then
     LOCATIONS_CSV=/
 fi
 
+NAXSI_LISTEN_PORT="10418"
```

Comment:
I think this would work.
@giles, @Joseph-Irving: Do we want to rebrand it as INTERNAL_LISTEN_PORT or similar?

There is some ugliness around the use of TLS on this. We could have it speak plan HTTP but we would need more complicated logic around the HTTP -> HTTPS redirect. Thoughts?